config/routes.rb

```diff
@@ -173,5 +173,9 @@ def put(*_args); end
     end
   end
 
+  namespace :cfme do
```

Comment:
To be publicly accessible, this needs to be somewhere inside the `api` scope.

Reply:
Gah you're right...forgot about that...